StackExchange.Profiling/BaseProfilerProvider.cs

```diff
@@ -35,6 +35,10 @@ public abstract class BaseProfilerProvider : IProfilerProvider
         /// </summary>
         public abstract MiniProfiler GetCurrentProfiler();
 
+        public abstract Timing GetHead();
```

Comment:
I'm not sure I understand why these methods are necessary.

Since the MiniProfiler.Current is stored in the exact same manner as Head. Seems like you could continue to use `MiniProfiler.Current.Head`?